src/components/NewMessage/NewMessageNewFileDialog.vue

```diff
@@ -210,11 +210,17 @@ export default {
 
 			let fileData
 			try {
-				const response = await createNewFile(
-					filePath,
-					this.selectedTemplate?.filename,
-					this.selectedTemplate?.templateType,
-				)
+				let response
+				if (this.selectedTemplate.fileid === -1) {
+					// Don't send template information for Blank
+					response = await createNewFile(filePath)
+				} else {
+					response = await createNewFile(
+						filePath,
+						this.selectedTemplate?.filename,
+						this.selectedTemplate?.templateType,
+					)
+				}
```

Comment:
What about a ternary operator here?

```suggestion
				const response = this.selectedTemplate.fileid === -1 
					? await createNewFile(filePath)
				  : await createNewFile(
						filePath,
						this.selectedTemplate?.filename,
						this.selectedTemplate?.templateType,
					)
```